src/meta/git_file_status.rs

```diff
@@ -0,0 +1,71 @@
+use crate::git::GitStatus;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct GitFileStatus {
+    pub index: GitStatus,
+    pub workdir: GitStatus,
+}
+
+impl Default for GitFileStatus {
+    fn default() -> Self {
+        Self {
+            index: GitStatus::Default,
+            workdir: GitStatus::Default,
+        }
+    }
+}
+
+impl GitFileStatus {
+    #[cfg(not(feature = "no-git"))]
+    pub fn new(status: git2::Status) -> Self {
+        Self {
+            index: match status {
+                s if s.contains(git2::Status::INDEX_NEW) => GitStatus::NewInIndex,
+                s if s.contains(git2::Status::INDEX_DELETED) => GitStatus::Deleted,
+                s if s.contains(git2::Status::INDEX_MODIFIED) => GitStatus::Modified,
+                s if s.contains(git2::Status::INDEX_RENAMED) => GitStatus::Renamed,
+                s if s.contains(git2::Status::INDEX_TYPECHANGE) => GitStatus::Typechange,
+                _ => GitStatus::Unmodified,
+            },
+
+            workdir: match status {
+                s if s.contains(git2::Status::WT_NEW) => GitStatus::NewInWorkdir,
+                s if s.contains(git2::Status::WT_DELETED) => GitStatus::Deleted,
+                s if s.contains(git2::Status::WT_MODIFIED) => GitStatus::Modified,
+                s if s.contains(git2::Status::WT_RENAMED) => GitStatus::Renamed,
+                s if s.contains(git2::Status::IGNORED) => GitStatus::Ignored,
+                s if s.contains(git2::Status::WT_TYPECHANGE) => GitStatus::Typechange,
+                s if s.contains(git2::Status::CONFLICTED) => GitStatus::Conflicted,
+                _ => GitStatus::Unmodified,
+            },
+        }
+    }
+
+    pub fn render(
+        &self,
+        colors: &crate::color::Colors,
+        git_symbols: &crate::git_symbol::GitSymbols,
+    ) -> crate::color::ColoredString {
+        use crate::color::{ColoredString, Colors};
```

Comment:
move `use` and `crate` to the top of the file

Reply:
Done for all in this file.